cli/src/main/java/de/jplag/cli/CliOptions.java

```diff
@@ -0,0 +1,142 @@
+import de.jplag.clustering.algorithm.InterClusterSimilarity;
+import de.jplag.options.JPlagOptions;
+import de.jplag.options.SimilarityMetric;
+
+import picocli.CommandLine;
+import picocli.CommandLine.ArgGroup;
+import picocli.CommandLine.Option;
+import picocli.CommandLine.Parameters;
+
+@CommandLine.Command(name = "jplag", description = "", usageHelpAutoWidth = true, abbreviateSynopsis = true)
+public class CliOptions implements Runnable {
+    public static final Language defaultLanguage = new de.jplag.java.Language();
+
+    @Parameters(paramLabel = "root-dirs", description = "Root-directory with submissions to check for plagiarism%n", split = ",")
+    public File[] rootDirectory = new File[0];
+
+    @Option(names = {"--new",
+            "-new"}, split = ",", description = "Root-directory with submissions to check for plagiarism (same as the root directory)%n")
+    public File[] newDirectories = new File[0];
+
+    @Option(names = {"--old", "-old"}, split = ",", description = "Root-directory with prior submissions to compare against%n")
+    public File[] oldDirectories = new File[0];
+
+    @Option(names = {"--language",
+            "-l"}, arity = "1", converter = LanguageConverter.class, completionCandidates = LanguageCandidates.class, description = "Select the language to parse the submissions (default: java). The language names are the same as the subcommands.%n")
+    public Language language = defaultLanguage;
+
+    @Option(names = {"-bc", "--bc",
+            "--base-code"}, description = "Path of  the  directory  containing  the  base  code  (common  framework  used  in  all submissions)%n")
+    public String baseCode;
+
+    @Option(names = {"-t", "--min-tokens"}, description = "Tunes the comparison sensitivity by adjusting the  minimum token required to be counted "
+            + "as a matching section. A smaller <n>  increases  the sensitivity but might lead to more " + "false-positives%n")
+    public Integer minTokenMatch = null;
+
+    @Option(names = {"-h", "--help"}, usageHelp = true, description = "display this help and exit")
+    public boolean help;
+
+    @Option(names = {"-n",
+            "--shown-comparisons"}, description = "The maximum number of comparisons that will  be  shown  in the generated report, if set "
+                    + "to -1 all comparisons will be shown (default: 100)%n")
+    public int shownComparisons = JPlagOptions.DEFAULT_SHOWN_COMPARISONS;
+
+    @Option(names = {"-r",
+            "--result-directory"}, description = "Name of the directory in which the comparison results will be stored (default: result)%n")
+    public String resultFolder = "results";
+
+    @ArgGroup(heading = "Advanced%n")
+    public Advanced advanced = new Advanced();
+
+    @ArgGroup(validate = false, heading = "Clustering%n")
+    public Clustering clustering = new Clustering();
+
+    /**
+     * Empty run method, so picocli prints help automatically
+     */
+    @Override
+    public void run() {
+    }
```

Comment:
If you move the comment into the method as a line comment, sonar will stop complaining.